bokehjs/src/lib/models/renderers/glyph_renderer.ts

```diff
@@ -251,8 +251,10 @@ export class GlyphRendererView extends DataRendererView {
           return this.model.view.convert_indices_from_subset(indices)
         else if (inspected.indices.length > 0)
           return inspected.indices
-        else
-          return map(Object.keys(inspected.multiline_indices), (i) => parseInt(i))
+        else {
+          // TODO: return inspected.multiline_indices.keys()
+          return Object.keys(inspected.multiline_indices).map((i) => parseInt(i))
```

Comment:
I hoped to replace this atrocity, but there is no good way to achieve this in a backwards compatible way. It would be if we could use proxies, but those are not available in IE and not polyfillable enough work here.

Reply:
I'd just like to state that I don't think comments like this are helpful, and especially the over the top language. Someone spent their time and effort to contribute and write that code and it provided value and benefit to our users, even if it is not perfect from a technical standpoint. And if there is anyone accountable for that lack, it is in fact us, since we decide what is merged. Please try to dial it back a little.